ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/custom-templates.j2

```diff
@@ -0,0 +1,10 @@
+{% if custom_templates is defined and custom_templates|length > 0 %}
```

Comment:
please refactor to: {% if custom_templates is defined and custom_templates is not none %}